src/sciline/task_graph.py

```diff
@@ -113,6 +113,19 @@ def compute(
         else:
             return self._scheduler.get(self._graph, [keys])[0]
 
+    def keys(self) -> Generator[Key, None, None]:
+        """Iterate over all keys of the graph.
+
+        Yields all keys, i.e., the types of values that can be computed or are
+        provided as parameters.
```

Comment:
Erm, this is odd, because we have `self._keys`, and the `keys` passed in the `__init__`, with a quite different meaning from what you implemented here.

Reply:
Renamed the args in other methods to `targets` to match the name in `transform_coords`.